packages/cart/src/models/line-item.ts

```diff
@@ -91,13 +94,13 @@ export default class LineItem {
   variant_option_values?: Record<string, unknown> | null
 
   @Property({ columnType: "boolean" })
-  requires_shipping = true
+  requires_shipping? = true
 
   @Property({ columnType: "boolean" })
-  is_discountable = true
+  is_discountable? = true
 
   @Property({ columnType: "boolean" })
-  is_tax_inclusive = false
+  is_tax_inclusive? = false
```

Comment:
nit: we have to define a pattern whether we use `?` for  properties that have a default value or not. hehe

Reply:
Should we have this convention:
```
@Property({ columnType: "boolean", default: false })
is_tax_inclusive = false
```

And require the prop to be included in the `OptionalProps`?

Reply:
I was thinking the same while reviewing the module payment
I think if a property is nullable or have a default value then it could always be
```ts
@Property({ columnType: "boolean", default: false })
is_tax_inclusive? = false

@Property({ columnType: "boolean", nullable: true })
is_tax_inclusive? = null
```
should also be added to the optionalProps
wdyt guys?

Reply:
To me, if the property is nullable and has a default value we do not use `?`

@Property({ columnType: "boolean", default: false })
prop = false

@Property({ columnType: "boolean", nullable: true })
prop?

@Property({ columnType: "boolean", nullable: true, default: false })
prop = false

Migrating to Mikro-ORM 6 we can also use [Opt](https://mikro-orm.io/docs/next/guide/relationships#alternative-approach-with-opt-type) to flag optional properties

Reply:
in the second example I would do

```ts
https://github.com/Property({ columnType: "boolean", nullable: true })
prop = null
```